scripts/install/install_unix.sh

```diff
@@ -823,6 +847,8 @@ main()
   if [ $# -ge 1 ]; then
     while [ -n "$1" ]; do
       case "$1" in
+        -q|--quiet)
```

Comment:
In my opinion, a quiet flag should not just remove the color of the ouput, but shrink the ouput at all completely or at least to a minimum, shouldn't it?

Reply:
A future revision of the scripts can account for full quiet where all non-error messages are suppressed. That will require a little refactoring that I'm not ready for due to the mess it will make of another open PR. I'll adjust description to account for that.

Reply:
Scratch that, I forgot we were overriding printf with our own function of it. I changed that function instead.